skiko/build.gradle.kts

```diff
@@ -16,8 +16,14 @@ plugins {
     id("de.undercouch.download") version "4.1.2"
 }
 
-val coroutinesVersion = "1.6.4"
-val atomicFuVersion = "0.18.5"
+val Project.supportWasm: Boolean
```

Comment:
This commit is going to be squashed if such way of separation of wasm build is ok.